modules/insights-operator-disabling-sca.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * support/remote_health_monitoring/insights-operator-simple-access.adoc
+
+
+[id="insights-operator-disabling-sca_{context}"]
+= Disabling Simple Content Access import
+
+You can disable the importation of RHEL Simple Content Access certs using the `support` secret in the `openshift-config` namespace.
```

Comment:
```suggestion
You can disable the importating of RHEL Simple Content Access certs using the `support` secret in the `openshift-config` namespace.
```

Changed importation to importing for translation purposes.

Reply:
Or perhaps "You can disable the import"

Reply:
One small nit also, I think 'certs' should be spelled out to 'certificates' for clarity.